scripts/scil_filter_streamlines_by_length.py

```diff
@@ -16,14 +22,20 @@ def _build_args_parser():
         description='Filter streamlines by length.')
     p.add_argument('in_tractogram', type=str,
                    help='Streamlines input file name.')
+
+    add_reference(p)
```

Comment:
This is technically shown as optional in the help (with a --). Should this be lower with the other ?